src/cc/bpf_module.cc

```diff
@@ -329,7 +329,7 @@ unique_ptr<ExecutionEngine> BPFModule::finalize_rw(unique_ptr<Module> m) {
 
 // load an entire c file as a module
 int BPFModule::load_cfile(const string &file, bool in_memory, const char *cflags[], int ncflags) {
-  clang_loader_ = make_unique<ClangLoader>(&*ctx_, flags_);
+  clang_loader_ = ebpf::make_unique<ClangLoader>(&*ctx_, flags_);
```

Comment:
Because the usage of `common.h` (which also has our version of `make_unique`) in more files lead to compiler complain about duplicated overload etc.